src/api/BikelyApi.js

```diff
@@ -1,5 +1,6 @@
 export class BikelyApi {
-  static apiUrl = 'https://coderscamp-bikely.herokuapp.com/';
+  static apiUrl = 'http://localhost:8080';
+  // static apiUrl = 'https://coderscamp-bikely.herokuapp.com/';
```

Comment:
Unnecessary comment, I think, but I know the purpose. By the way, leaving the slash at the end of the Uri/Url brings some troubles. It depends, of course, on how the particular controller has been configured and we need, maybe, some unified approach where this slash should be provided when needed.

Reply:
Ok, comment and slash removed.